jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -313,12 +318,9 @@ node {
             stage('Product Version Update') {
                 when(isGARelease) {
                     gitCommitWhenChanges("Updated ${productName} product version to ${latestRelease}") { msgs ->
-                        manifestFileTxt = readFile(installationManifestFile)                        
-                        // Updates the release tag version
-                        manifestFileTxt = manifestFileTxt.replaceFirst(/${releaseTagVar}: '.*'/, "${releaseTagVar}: '${latestRelease}'")
-                        
-                        // Updates the product version
-                        manifestFileTxt = manifestFileTxt.replaceFirst(/${productVersionVar}: '.*'/, "${productVersionVar}: '${productVersion}'")
+                        manifestFileTxt = readFile(installationManifestFile)
+                        updateManifestVariable(manifestFileTxt, releaseTagVar, latestRelease)
+                        updateManifestVariable(manifestFileTxt, productVersionVar, productVersion)
```

Comment:
updateManifestVariable will check that both name and value exist so we are safe to just call both of these in all scenarios.